sdk/documenttranslation/azure-ai-documenttranslation/setup.py

```diff
@@ -0,0 +1,70 @@
+from setuptools import setup, find_packages
+import os
+from io import open
+import re
+
+PACKAGE_NAME = "azure-ai-documenttranslation"
+PACKAGE_PPRINT_NAME = "Document Translation Package"
+
+# a-b-c => a/b/c
+package_folder_path = PACKAGE_NAME.replace('-', '/')
+# a-b-c => a.b.c
+namespace_name = PACKAGE_NAME.replace('-', '.')
+
+# Version extraction inspired from 'requests'
+with open(os.path.join(package_folder_path, '_version.py'), 'r') as fd:
+    version = re.search(r'^VERSION\s*=\s*[\'"]([^\'"]*)[\'"]',
+                        fd.read(), re.MULTILINE).group(1)
+if not version:
+    raise RuntimeError('Cannot find version information')
+
+with open('README.md', encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name=PACKAGE_NAME,
+    version=version,
+    description='Microsoft Azure {} Client Library for Python'.format(PACKAGE_PPRINT_NAME),
+
+    # ensure that these are updated to reflect the package owners' information
+    long_description=long_description,
+    url='https://github.com/Azure/azure-sdk-for-python',
+    author='Microsoft Corporation',
+    author_email='azuresdkengsysadmins@microsoft.com',
```

Comment:
didn't know this was a thing

Reply:
This was just copied straight from our template package